src/agor/tools/agent_coordination.py

```
@@ -23,6 +23,8 @@
 from pathlib import Path
 from typing import Dict, List, Optional
 
+from .strategy_config import StrategyConfigManager
```

Pylint (3.3.7)

[error] Attempted relative import beyond top-level package

(E0402)